hr_timesheet_sheet/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Eficent
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'HR Timesheet Sheet',
+    'version': '12.0.1.0.0',
+    'category': 'Human Resources',
+    'sequence': 80,
+    'summary': 'Timesheet Sheets, Activities',
+    'license': 'AGPL-3',
+    'author': (
+        'Eficent, '
+        'Odoo Community Association (OCA)'
+    ),
+    'website': 'https://github.com/OCA/hr-timesheet',
+    'depends': [
+        'hr_timesheet',
+        'web_widget_x2many_2d_matrix',
+    ],
+    'data': [
```

Comment:
`data` sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)
If XML `group` not use CSV then LGTM

Reply:
Adjusted as per template